docs/docs/chitchat-faqs.mdx

```diff
@@ -341,90 +77,112 @@ pipeline:
     epochs: 100
 ```
 
-Now that we've defined the message handling side, we need to make
-the dialogue handling parts aware of these changes. First, we need to add the
-new intents to our domain:
-
-```yaml-rasa title="domain.yml"
-intents:
-  - greet
-  - bye
-  - thank
-  - faq
-```
-
-We'll also need to add a [retrieval action](./retrieval-actions.mdx),
-which takes care of sending the response predicted from the `ResponseSelector`
-back to the user, to the list of actions. These actions always have to start
-with the `respond_` prefix:
+By default, the `ResponseSelector` will build a single retrieval model for all retrieval intents.
+If you want to retrieve responses for FAQs and chitchat separately, you can use multiple `ResponseSelector` components
+and specify the `retrieval_intent` key, for example:
 
-```yaml-rasa title="domain.yml"
-actions:
-  - respond_faq
-```
-
-Next we'll write a story so that the dialogue engine knows which action to predict:
-
-```yaml-rasa title="data/stories.yml"
-stories:
-- story: Some question from FAQ
-  steps:
-  - intent: faq
-  - action: respond_faq
+```rasa-yaml
+pipeline:
+# Other components
+- name: ResponseSelector
+  epochs: 100
+  retrieval_intent: faq
+- name: ResponseSelector
+  epochs: 100
+  retrieval_intent: chitchat
 ```
 
-This prediction is handled by the `MemoizationPolicy`, as we described earlier.
+It's recommended to start with separate models for each `retrieval_intent`. You can merge them later if you find it works better for your use case.
 
-### 4. Using the bot
 
-After all of the changes are done, train a new model and test the modified FAQs:
+### 3. Creating rules
 
-```bash
-rasa train
-rasa shell
-```
+When using retrieval actions, you need to write only one rule for each retrieval intent. All intents
```

Comment:
this may be starting to get a little too technical? Maybe we can do this more from a "group of FAQs" perspective rather than saying "when using retrieval actions" all the time